packages/x-date-pickers/src/tests/describeValue/testControlledUnControlled.tsx

```diff
@@ -66,10 +66,25 @@ export const testControlledUnControlled: DescribeValueTestSuite<any, any> = (
     it('should call onChange when updating a value defined with `props.value`', () => {
       const onChange = spy();
 
-      const { selectSection } = renderWithProps({ defaultValue: values[0], onChange });
+      const useControlledElement = (props) => {
```

Comment:
This test does not make sense without wrapper controlling state as it breaks in case of components re-rendering on view change (i.e. TimeClock) and this behavior has been uncovered after adding testing with the default views instead of only testing with one view.
If we don't want this custom behavior, we can ditch this test altogether, because it is now just a duplicate of the test above and no longer matches the description... 🙈

Reply:
>  because it is now just a duplicate of the test above 

But using `value` instead of `defaultValue` if I'm not mistaken

Reply:
Am I missing something?
https://github.com/mui/mui-x/blob/master/packages/x-date-pickers/src/tests/describeValue/testControlledUnControlled.tsx#L48
It looks like the only difference between this and the below test (the one I changed) is the omission of the `assertRenderedValue` call in the second test. 🤷

Reply:
Could you give me the title of the 2 tests that are redundant for you?
Because I'm looking at

1. should call onChange when updating a value defined with `props.defaultValue` and update the rendered value
=> uses `defaultValue`

2. `should call onChange when updating a value defined with `props.value``
=> uses `value`

Reply:
Yes, those are the exact tests that I'm talking about.
Correct me if I'm wrong, but on master, they both use `defaultValue`, that's why I'm saying that either we adjust them as I did or just remove one of those as they are no longer doing what the title states... 🤷

Reply:
OK you were talking about adjusting the test compared to master
I thought you were saying that after your modification this test was becoming a duplicate.

This is just a typo from https://github.com/mui/mui-x/pull/9009/files#diff-5cdf0facdbadac7fca4b02cd2f21e4e274e2911f215483ca1b9fe0906faf3bebR69, no problem to use `value` again.

Reply:
Ok, we are in the same boat then.
So, just clarifying, simply using `value` doesn't work anymore, because this test relied on all components having only one `view`. I've removed the override of only a single view for certain tests and it started failing at that time with both Digital and Analog clocks. 🙈